### ISSUE TITLE
MiloLocal.lua: 222: Attempt to call field 'setStatus' (a nil value) - after yesterdays update!

### ISSUE THREAD

**Issue by DistantT** [kepler155c/opus-apps#41]
After your update (on the 26th? July 2020) Milo fails to run with this error:

![Distant-2020-07-25_19 45 34-Cropped](https://user-images.githubusercontent.com/68794113/88467479-7f72bf00-cecf-11ea-8531-4749c3f749e2.png)
![Distant_2020-07-25_23 18 57_Cropped](https://user-images.githubusercontent.com/68794113/88467483-87cafa00-cecf-11ea-8b73-f7bd5cda4ea6.png)

**Comment by kepler155c** [comment]
It looks like the turtle package may not have initialized correctly. I pushed a change to (hopefully) show a better error message - so we can get a better idea of what is failing. You can run System->Update to get the latest.

**Comment by DistantT** [comment]
I ran update from the System tab:
So more detail ....
I haven't looked at the code at all.... 
![Distant-2020-07-26_12 41 57-Cropped](https://user-images.githubusercontent.com/68794113/88478193-0199cc80-cf3e-11ea-8f6c-a4b2bac3741c.png)

**Comment by xAnavrins** [comment]
Install/Update the turtle package.

**Comment by DistantT** [comment]
OK Yes success. I hadn't installed the turtle package on this Milo system, as it wasn't previously needed..... So now I have added(installed) it, and after a reboot, Milo seems to be working perfectly, with machines crunching and smelting automatically, and things going in and out of storage etc.... so it looks to be solved. [s]I `think will dare`[/s] **I have now dared** to update my main storage Milo now! If the turtle package is now needed for Milo should it be a dependency? Yes that Milo Turtle already had the turtle package and there is no error... I think this is closed from my side...  TYVM
